st2actions/st2actions/runners/actionchainrunner.py

```diff
@@ -63,6 +67,35 @@ def init_vars(self, action_parameters):
             self.vars = self._get_rendered_vars(self.actionchain.vars,
                                                 action_parameters=action_parameters)
 
+    def validate(self):
```

Comment:
Similar as below - another option is to move this to the ActionChain class (it doesn't require any run time info) and validate the workflow and throw during resource loading phase (meaning when user runs register-actions).

What do other people think?